pkg/crc/cluster/cluster.go

```diff
@@ -93,7 +93,7 @@ func AddPullSecret(sshRunner *ssh.SSHRunner, ocConfig oc.OcConfig, pullSec strin
 		fmt.Sprintf(`{"data":{".dockerconfigjson":"%s"}}`, base64OfPullSec),
 		"-n", "openshift-config", "--type", "merge"}
 
-	if err := oc.WaitForOpenshiftResource(ocConfig, "secret"); err != nil {
+	if err := WaitForOpenshiftResource(ocConfig, "secret"); err != nil {
```

Comment:
This can be made private to the package after these changes (but not sure it should be private as it can be useful outside of the cluster package)

Reply:
yes, I let this function public.